src/makeCalculator.js

```diff
@@ -4,7 +4,31 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {};
+
+  calculator.result = 0;
+
+  calculator.operate = (callback, number) => {
+    calculator.result = callback(number);
+
+    return calculator;
+  };
+
+  calculator.add = (a) => calculator.result + a;
+
+  calculator.subtract = (a) => calculator.result - a;
+
+  calculator.multiply = (a) => calculator.result * a;
+
+  calculator.divide = (a) => calculator.result / a;
```

Comment:
The `add`, `subtract`, `multiply`, `divide` methods are not modifying the `result` property directly. They should be like this: `calculator.result += a;` for `add`, `calculator.result -= a;` for `subtract`, `calculator.result *= a;` for `multiply`, `calculator.result /= a;` for `divide`. This is a CODE STYLE issue, as you should use arithmetic operators with assignment operator `=` if you want to add something to an existing value.